server.py

```diff
@@ -0,0 +1,32 @@
+from fastapi import FastAPI, Response, status
+from pydantic import BaseModel
+from config import settings
+from utils import ModelName, selectModel, send_rest_api_req
+
+class Prompt(BaseModel):
+    question: str
+    context: str
+    model: ModelName = ModelName.distilbert
+
+app = FastAPI()
+
+@app.get("/")
+def home_route():
+    return {"Message": "welcome to bons.ai"}
+
+@app.post("/prompt")
+async def prompt_llm(prompt: Prompt, response: Response):
+    context = prompt.context
```

Comment:
Its not typical to require the Response object in as a parameter Aaroosh. Was there a reason this has been added? 

On researching I found this https://fastapi.tiangolo.com/advanced/response-change-status-code/#use-a-response-parameter so its good research from you too. I have just used a different pattern to do this. But its okay here as its documented by the library.